app/build.gradle

```diff
@@ -5,25 +5,32 @@ android {
     buildToolsVersion "25.0.2"
     defaultConfig {
         applicationId "com.edu.uz.currency.currencyapp"
-        minSdkVersion 21
+        minSdkVersion 19
```

Comment:
W tym pliku musiałem pozmieniać te kilka rzeczy, bo inaczej mi się nie kompiluje. Sprawdźcie czy u was to działa, żebym czegoś nie zepsuł.